docs/source/installation.mdx

```diff
@@ -16,183 +16,189 @@ limitations under the License.
-```
-~/anaconda3/envs/main/lib/python3.7/site-packages/
-```
-now this editable install will reside where you clone the folder to, e.g. `~/transformers/` and python will search it too.
+These commands will link the folder you cloned the repository to and your Python library paths. Python will now look inside this folder in addition to the normal library paths. For example, if your Python packages are typically installed in `~/anaconda3/envs/main/lib/python3.7/site-packages/`, Python will also search the folder you cloned to: `~/transformers/`.
 
-Do note that you have to keep that `transformers` folder around and not delete it to continue using the  `transformers` library.
+<Tip warning={true}>
+You must keep the `transformers` folder if you want to keep using the library.
+</Tip>
 
-Now, let's get to the real benefit of this installation approach. Say, you saw some new feature has been just committed into `master`. If you have already performed all the steps above, to update your transformers to include all the latest commits, all you need to do is to `cd` into that cloned repository folder and update the clone to the latest version:
+Now you can easily update your clone to the latest version of 🤗 Transformers with the following commands:
 
-```
+```bash
 cd ~/transformers/
 git pull
 ```
 
-There is nothing else to do. Your python environment will find the bleeding edge version of `transformers` on the next run.
-
+Your Python environment will find the `master` version of 🤗 Transformers on the next run.
 
-## With conda
+## Install with conda
 
-Since Transformers version v4.0.0, we now have a conda channel: `huggingface`.
+Install from the conda channel `huggingface`:
 
-🤗 Transformers can be installed using conda as follows:
-
-```
+```bash
 conda install -c huggingface transformers
 ```
 
-Follow the installation pages of TensorFlow, PyTorch or Flax to see how to install them with conda.
-
-## Caching models
-
-This library provides pretrained models that will be downloaded and cached locally. Unless you specify a location with
-`cache_dir=...` when you use methods like `from_pretrained`, these models will automatically be downloaded in the
-folder given by the shell environment variable ``TRANSFORMERS_CACHE``. The default value for it will be the Hugging
-Face cache home followed by ``/transformers/``. This is (by order of priority):
+## Cache setup
 
-  * shell environment variable ``HF_HOME``
-  * shell environment variable ``XDG_CACHE_HOME`` + ``/huggingface/``
-  * default: ``~/.cache/huggingface/``
+Pretrained models are downloaded and locally cached at: `~/.cache/huggingface/transformers/`. This is the default directory given by the shell environment variable `TRANSFORMERS_CACHE`. You can change the shell environment variables shown below - in order of priority - to specify a different cache directory:
```

Comment:
This is only true for linux/mac. It would be nice to also put that for Windows - do you have access to a Windows machine?

Reply:
Good idea, and yes I have a Windows machine! Would the default directory on Windows be `C:\Users\username\.cache\huggingface\transformers`?

Reply:
That's why I'm asking, I don't know :)